apps/betterangels-backend/accounts/tests/test_models.py

```diff
@@ -54,6 +54,19 @@ def test_is_outreach_authorized(
         remove_organization_permission_group(unauthorized_org)
         self.assertEqual(user.is_outreach_authorized, should_succeed)
 
+    def test_display_pronouns(self) -> None:
+        client_profile = baker.make(ClientProfile)
+        self.assertIsNone(client_profile.display_pronouns)
+
+        client_profile.pronouns = PronounEnum.HE_HIM_HIS
+        client_profile.save()
+        self.assertEqual(client_profile.display_pronouns, PronounEnum.HE_HIM_HIS.label)
```

Comment:
```suggestion
        self.assertEqual(client_profile.display_pronouns, "he/him/his")
```
just because otherwise if `.label` were coming back as `None` for some reason, this test wouldn't fail since you're using that method in the code AND in the test

Reply:
good catch